css-source/components/_debug-table.scss

```diff
@@ -0,0 +1,28 @@
+.c-debug-table td {
+    font-family: Monaco, Consolas, monospace;
+}
+
+/* X column tables have the 2nd+ cell right aligned */
+.c-debug-table th:nth-child(n+2),
+.c-debug-table td:nth-child(n+2) {
+    text-align: right;
+}
+
+.c-debug-table td:nth-child(n+2) {
+    word-wrap: break-word;
+    word-break: break-all;
+    word-break: break-word; /* Not standard for webkit */
+}
+
+.c-debug-table th.left,
+.c-debug-table td.left {
+    text-align: left;
+}
+
+.c-debug-table .duplicate-route td {
+    background: #ffea96;
```

Comment:
Should we be using CSS custom properties for colors?

Reply:
I will introduce CSS Variables when removing the SCSS code